drivers/FileSR.py

```diff
@@ -707,7 +707,7 @@ def reset_leaf(self, sr_uuid, vdi_uuid):
         vhdutil.killData(self.path)
 
     def _do_snapshot(self, sr_uuid, vdi_uuid, snap_type,
-                     secondary=None, cbtlog=None):
+                     _=False, secondary=None, cbtlog=None):
```

Comment:
This just makes the method signature match the method it is overriding.

Reply:
Since Python doesn’t strictly enforce matching method signatures in overrides, so maybe it`s not necessary or using `*args` could be a more flexible approach. This way, we can handle additional parameters without needing to care about their both the key and value :-)